libs/data-mapper/src/lib/utils/__test__/DataMapUtils.spec.ts

```diff
@@ -20,6 +23,35 @@ import {
 } from '../__mocks__';
 
 describe('utils/DataMap', () => {
+  describe('addAncestorNodesToCanvas', () => {
+    const sourceSchema: Schema = layeredLoopSourceMockSchema;
+    const extendedSourceSchema: SchemaExtended = convertSchemaToSchemaExtended(sourceSchema);
+    const flattenedSchema = flattenSchemaIntoDictionary(extendedSourceSchema, SchemaType.Source);
+    it('includes direct parent', () => {
```

Comment:
Nit: some new lines here and below to break the `it()` from each other

Reply:
done!